File: C_code/prime.c
#include<stdio.h>
int main()
{
	int num,flag=0,i;
	printf("enter a number:");
	scanf("%d",&num);
	for(i=2;i<num;i++)
	{
		if(num%i==0)
		{
			flag=1;
			break;
		}
		else flag=0;
	}
	if(flag==1)
	{
		printf("given number %d is not a prime number\n",num);
	}
	else printf("given number %d is prime number\n",num);
}

File: Python/prime.py
number=int(input("enter the number:"))
flag=0
for i in range(2,number//2):
    if(number%i)==0:
        flag=1
        break
    else:
        flag=0
if flag==1:
    print(number,"is not a prime number")
else:
    print(number,"is a prime number")
